src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Header} from './components/Header/Header';
```

Comment:
Not an error, but this kind of imports looks weird. You can use index file, to simplify them to this:
```js
import {Header} from './components/Header';
```

and your `index` file can be smth like this:
```js
export { Component } from './Component';
```